grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -2239,15 +2241,18 @@ def section_hummock_manager(outer_panels):
                     ],
                 ),
                 panels.timeseries_kilobytes(
-                    "Table KV Size",
+                    "Table Size",
                     "",
                     [
                         panels.target(f"{table_metric('storage_version_stats', total_key_size_filter)}/1024",
                                       "table{{table_id}} {{metric}}"),
                         panels.target(f"{table_metric('storage_version_stats', total_value_size_filter)}/1024",
                                       "table{{table_id}} {{metric}}"),
+                        panels.target("sum by (table_id) (storage_version_stats{metric='total_key_size',table_id=~\"$table\",job=~\"$job\",instance=~\"$node\"}) / 1024 + sum by (table_id) (storage_version_stats{metric='total_value_size',table_id=~\"$table\",job=~\"$job\",instance=~\"$node\"}) / 1024", "table size - {{table_id}}"),
```

Comment:
Let;s use the `table_metric` function to generate the metric with filter instead of writing out the filter directly. `table_metric` function can ensure that the metric will work in both the cloud deployment and local deployment panel.

Reply:
Got it.